packages/theme-chalk/src/transfer.scss

```diff
@@ -16,14 +16,6 @@
   }
 
   @include e(button) {
-    display: block;
-    margin: 0 auto;
-    padding: 10px;
-    border-radius: 50%;
-    color: $--color-white;
-    background-color: $--color-primary;
-    font-size: 0;
-
     @include when(with-texts) {
       border-radius: $--border-radius-base;
     }
```

Comment:
Seems like the rest part is also invalid, could you delete them all?

Reply:
sure, It's excatly a useless pattern, I'll remove it.